src/components/Create/BudgetForm.tsx

```diff
@@ -75,15 +91,15 @@ export default function BudgetForm({
                 setShowFundingFields(checked)
               }}
             />
-            <label>Set a funding target</label>
+            <label>Set a funding goal</label>
           </Space>
         </Form.Item>
 
         {target === maxIntStr && (
           <p style={{ color: colors.text.primary }}>
-            <span style={{ fontWeight: 600 }}>No target:</span> All funds can be
-            distributed by the project, and the project will have no overflow.
-            (This is the same as setting the target to infinity.)
+            <span style={{ fontWeight: 600 }}>If your goal is 0:</span> The
+            entire project's funds can be distributed. There will be no
+            overflow. This is the same as setting the goal to infinity.
```

Comment:
This is not the case.

If your goal is 0, all funds are overflow and can be accessible by token holders by burning tokens.
If goal is off (which sets it to infinity), all of the project's funds can be distributed to the project.